Exercise_B/index.html

```diff
@@ -19,16 +25,81 @@
 
   <body>
     <!--Header (flower shop logo block element)-->
+    <header>
+      <div class="logo container">
+        <img src="assets/images/logo.png" alt="Logo of the shop">
+      </div>
+    </header>
     <!--Banner Jumbotron of the image-->
+  
+    <div class="jumbotron" style="background-image:
+     linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)), 

```

Comment:
If you do not want you background image to repeat itself:
.hero-image {
  background-image: url("photographer.jpg"); /* The image used */
  background-color: #cccccc; /* Used if the image is unavailable */
  height: 500px; /* You must set a specified height */
  background-position: center; /* Center the image */
  background-repeat: no-repeat; /* Do not repeat the image */
  background-size: cover; /* Resize the background image to cover the entire container */
}
https://www.w3schools.com/cssref/pr_background-repeat.asp